packages/terra-table/CHANGELOG.md

```diff
@@ -4,11 +4,13 @@
 
 * Breaking Changes
   * Renamed `hasColumnHeaders` prop to `hasVisibleColumnHeaders`.
+  * Renamed `hasSelectableRows` prop to `rowSelectionMode` with an enumeration.
```

Comment:
Do we have an upgrade guide for these breaking changes?

Reply:
We have not done a 1.0 release yet so we should not need an upgrade guide.